tests/system/feature_store/test_spark_engine.py

```diff
@@ -0,0 +1,78 @@
+# Marked as enterprise because of v3io mount and pipelines
+import os
+
+import pytest
+
+import mlrun.feature_store as fs
+from mlrun import store_manager
+from mlrun.datastore.sources import ParquetSource
+from tests.system.base import TestMLRunSystem
+
+
+@TestMLRunSystem.skip_test_if_env_not_configured
+@pytest.mark.enterprise
+class TestFeatureStoreSparkEngine(TestMLRunSystem):
+    project_name = "fs-system-spark-engine"
+    spark_service = ""
+    pq_source = "testdata.parquet"
+
+    @classmethod
+    def _init_env_from_file(cls):
+        env = cls._get_env_from_file()
+        cls.spark_service = env["DEFAULT_SPARK_SERVICE"]
+        os.environ.setdefault("V3IO_USERNAME", env["V3IO_USERNAME"])
+
+    def get_local_pq_source_path(self):
+        return os.path.relpath(str(self.assets_path / self.pq_source))
+
+    def get_remote_pq_source_path(self, without_prefix=False):
+        path = "v3io://"
+        if without_prefix:
+            path = ""
+        path += "/bigdata/" + self.pq_source
+        return path
+
+    def custom_setup(self):
+        from mlrun import get_run_db
+        from mlrun.run import new_function
+        from mlrun.runtimes import RemoteSparkRuntime
+
+        if not self.spark_service:
```

Comment:
If not spark service then this test should fail no ?

Reply:
No, this is me just making sure that the image deployment is executed once (as I cannot do it in the setup_class due to the way mlrun tests are built).